start.py

```diff
@@ -715,6 +734,43 @@ def _configure_kudu(deployment, cluster, kudu_version):
             # Only one parcel can be activated at a time, so once one is found, our work is done.
             break
 
+
+def _validate_kudu_version(kudu_version, cdh_version, cm_cluster):
+    cdh_version_tuple = tuple([int(i) for i in cdh_version.split('.')])
+
+    if cdh_version_tuple < EARLIEST_CDH_VERSION_WITH_KUDU:
+        cdh_version_from_map = KUDU_PARCEL_VERSIONS.get(kudu_version)
+        if cdh_version_from_map is None:
+            raise Exception('kudu-version {} is not available for '
```

Comment:
Nit `kudu-version` => `Kudu version`? (here and below)

Reply:
Done.